src/init.ts

```diff
@@ -138,7 +138,9 @@ export function init(
     return function rmCb() {
       if (--listeners === 0) {
         const parent = api.parentNode(childElm) as Node;
-        api.removeChild(parent, childElm);
+        if (parent) {
```

Comment:
```suggestion
        if (parent !== null) {
```

Just to be more explicit about what is checked here.

Reply:
thank you, the suggested change is amended to the commit